<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VegetablePrintWidget</class>
 <widget class="QWidget" name="VegetablePrintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>20</width>
     <height>651</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>631</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>541</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="prePrintBtn">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="printBtn">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>打印</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="printArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QLabel" name="solderLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>551</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>541</width>
      <height>17</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="numberNameLabel">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>总量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberLabel">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pricesNameLabel">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>总计：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="priceLabel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0.0元</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dateNameLabel">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>日期：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataLabel">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>XXXX-XX-XX</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>541</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="addrLabel">
       <property name="text">
        <string>地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="addrTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>prePrintBtn</tabstop>
  <tabstop>printBtn</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
